editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -6003,6 +6069,16 @@ CanvasItemEditor::CanvasItemEditor(EditorNode *p_editor) {
 	skeleton_menu->get_popup()->set_item_checked(skeleton_menu->get_popup()->get_item_index(SKELETON_SHOW_BONES), true);
 	singleton = this;
 
+	// To ensure that scripts can parse the list of shortcuts correctly, we have to define
+	// those shortcuts one by one.
+	// Resetting zoom to 100% is a duplicate shortcut of `canvas_item_editor/reset_zoom`,
+	// but it ensures both 1 and Ctrl + 0 can be used to reset zoom.
```

Comment:
<kbd>Ctrl + 0</kbd> could probably be removed after this? I think <kbd>Ctrl + 0</kbd> is less convenient to reach than <kbd>1</kbd> since it requires both hands.

Reply:
Oh, this was already discussed here, whoops: https://github.com/godotengine/godot/pull/48252#issuecomment-876635566